src/drunc/session_manager/interface/session_manager.py

```diff
@@ -35,9 +39,22 @@ def serve(session_manager: SessionManager, address: str) -> None:
 # )
 # def session_manager_cli(log_level: str, log_path: str):
 def session_manager_cli():
+    app_name = "session_manager"
     log_level = "DEBUG"
+
     setup_root_logger(log_level)
-    logger = get_logger(logger_name="drunc.session_manager", rich_handler=True)
+    # log_path = get_log_path(
+    #     user=getpass.getuser(),
+    #     session_name=pmConfFileName,
+    #     application_name=appName,
+    #     override_logs=override_logs,
+    #     app_log_path=log_path,
+    # )
+    logger = get_logger(app_name)
+    create_logger_handler(
+        # log_file_path="",
+        rich_handler=True,
+    )
```

Comment:
I'd remove all of these commented lines
```suggestion
    logger = get_logger(app_name)
    create_logger_handler(rich_handler=True)
```